public/entity2/concreteentitylist.h

```diff
@@ -0,0 +1,28 @@
+#ifndef CONCRETEENTITYLIST_H
+#define CONCRETEENTITYLIST_H
+
+#include "entityidentity.h"
+
+class CConcreteEntityList
+{
+	struct CList
+	{
+		CEntityIdentity* m_pHead;
+		CEntityIdentity* m_pTail;
+		uint64_t unknown;
+	};
+public:
+	CEntityIdentities *m_pIdentityChunks[MAX_ENTITY_LISTS];
```

Comment:
This param could be replaced with ``CEntityIdentity *m_pIdentityChunks[MAX_ENTITY_LISTS];`` thus removing the ``CEntityIdentities`` redundant class